build.sh

```diff
@@ -258,8 +258,8 @@ function buildVP4620Image {
 
 	cp build/coreboot.rom protectli_$1_DF_$version.rom
```

Comment:
shouldn't this also be
```suggestion
	cp build/coreboot.rom protectli_vault_cml_$version.rom
```
?

Reply:
Yes, thanks. Fixed.